Sources/SourceKitLSP/CapabilityRegistry.swift

```diff
@@ -12,14 +12,17 @@
 
 import LSPLogging
 import LanguageServerProtocol
-
-/// Handler responsible for registering a capability with the client.
-public typealias ClientRegistrationHandler = (CapabilityRegistration) -> Void
+import SKSupport
 
 /// A class which tracks the client's capabilities as well as our dynamic
 /// capability registrations in order to avoid registering conflicting
 /// capabilities.
-public final class CapabilityRegistry {
+public final actor CapabilityRegistry {
+  /// The client's capabilities as they were reported when sourcekit-lsp was launched.
+  public let clientCapabilities: ClientCapabilities
```

Comment:
Maybe `staticClientCapabilities`? Or `initialClientCapabilities` or... something.

Reply:
There is no dynamic registration of client capabilities, so it doesn’t make sense to differentiate them.